.github/ISSUE_TEMPLATE/rollout-issue.md

```diff
@@ -0,0 +1,68 @@
+---
+name: Rollout issue
+about: 'Create issue for arcade-services rollout'
+title: Rollout DD-MM-YYYY
+labels: Rollout
+assignees: ''
+
+---
+
+# Purpose
+
+This issue tracks the `arcade-services` repository rollout. On top of the [Rollout](https://dev.azure.com/dnceng/internal/_wiki/wikis/DNCEng%20Services%20Wiki/831/Rollout) instructions described on the wiki, it provides the person responsible for the rollout checklist of the steps that should to be performed to rollout services in this repository. All relevant information, including the rollout PR, issues encountered during the rollout and steps taken to resolve them should be linked or added to this issue to keep full audit trail of changes rolled out to production.
+
+# Process
+
+## Build status check (Monday)
+- [ ] Check the status of the [dotnet-arcade-services-weekly](https://dev.azure.com/dnceng/internal/_build?definitionId=993) pipeline
+- [ ] Ensure the [arcade-services-internal-ci](https://dev.azure.com/dnceng/internal/_build?definitionId=252) pipeline is green
+- [ ] In case there is a problem with the CI build, notify the [Rollout channel](https://teams.microsoft.com/l/channel/19%3a72e283b51f9e4567ba24a35328562df4%40thread.skype/Rollout?groupId=147df318-61de-4f04-8f7b-ecd328c256bb&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47)
+
+## Rollout preparation (Tuesday)
+- [ ] Wait for the vendor to prepare the rollout. This includes a thread on the [Rollout channel](https://teams.microsoft.com/l/channel/19%3a72e283b51f9e4567ba24a35328562df4%40thread.skype/Rollout?groupId=147df318-61de-4f04-8f7b-ecd328c256bb&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47), rollout issue in [AzDO](https://dev.azure.com/dnceng/internal/_workitems/) and the rollout PR
+- [ ] Link the rollout PR to the "Rollout data" section of this issue
+- [ ] Double-check that the release notes contain all information
+- [ ] Merge the already prepard rollout PR
+- [ ] Ensure the build is green and stops on the "Approval" phase
+
```

Comment:
```suggestion
- [ ] Send an email to **.NET Core Engineering Partners** with the planned rollout date and release notes.

```

@tkapin do we want this?

Reply:
I see we reference the email later in the rollback section so I presume we do?

Reply:
Let's sync on this tomorrow again. I believe we agreed on these emails not having much value, but I'd like to check with you again on this.